apps/style/applab/style.scss

```diff
@@ -209,18 +210,56 @@ div#visualizationResizeBar {
 
 // ** END OF RULES THAT ARE MODIFIED AT RUNTIME IN adjustAppSizeStyles() **
 
-#divApplab {
+#divApplab, #visualizationOverlay {
   overflow: hidden;
   outline: none;
+  -webkit-touch-callout: none;
+  @include user-select(none);
+}
+
+#divApplab {
   position: relative;
   z-index: 1;
   background-color: white;
-  -webkit-touch-callout: none;
-  -webkit-user-select: none;
-  -khtml-user-select: none;
-  -moz-user-select: none;
-  -ms-user-select: none;
-  user-select: none;
+}
+
+#visualizationOverlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  z-index: 2;
+  pointer-events: none;
+
+  & > svg {
+    width: 100%;
+    height: 100%;
+    pointer-events: none;
+
+    .crosshair {
+      line.cursor {
+        stroke: #333;
+        stroke-width: 2;
+      }
+
+      line.guide {
+        stroke: #aaa;
+        stroke-width: 1.8;
+        stroke-dasharray: 6,5
+      }
+
+      text {
+        text-anchor: middle;
+
+        &.outline {
+          stroke: white;
+          stroke-width: 10;
+          stroke-linecap: round;
+          stroke-linejoin: round;
+          fill: none;
+        }
+      }
+    }
+  }
```

Comment:
In general, our strategy has been to get as much as we can as inline styles within the React component rather than as CSS. I think most, if not all, of this should be able to be moved.